src/main/java/net/neoforged/neoforge/event/entity/item/ItemExpireEvent.java

```diff
@@ -6,32 +6,33 @@
 package net.neoforged.neoforge.event.entity.item;
 
 import net.minecraft.world.entity.item.ItemEntity;
-import net.neoforged.bus.api.ICancellableEvent;
 
 /**
  * Event that is fired when an EntityItem's age has reached its maximum
- * lifespan. Canceling this event will prevent the EntityItem from being
- * flagged as dead, thus staying it's removal from the world. If canceled
- * it will add more time to the entities life equal to extraLife.
+ * lifespan. Adding extra life time will prevent the EntityItem from being
+ * flagged as dead, thus staying it's removal from the world.
  */
-public class ItemExpireEvent extends ItemEvent implements ICancellableEvent {
-    private int extraLife;
+public class ItemExpireEvent extends ItemEvent {
+    private int extraLife = 0;
 
     /**
      * Creates a new event for an expiring EntityItem.
      * 
      * @param entityItem The EntityItem being deleted.
-     * @param extraLife  The amount of time to be added to this entities lifespan if the event is canceled.
      */
-    public ItemExpireEvent(ItemEntity entityItem, int extraLife) {
+    public ItemExpireEvent(ItemEntity entityItem) {
         super(entityItem);
-        this.setExtraLife(extraLife);
     }
 
     public int getExtraLife() {
         return extraLife;
     }
 
+    /**
+     * Sets the amount of extra life time (in ticks) to give this EntityItem.
+     * 
+     * @param extraLife The amount of time to be added to this entities lifespan.
+     */
     public void setExtraLife(int extraLife) {
```

Comment:
Potentially silly suggestion. How about adding `addExtraLife` to make it clear to modders that they can add onto other people's extra life instead of overwriting? People might forget that is an option when doing set

Reply:
Not so silly at all, I also was contemplating adding addExtraLife(). It just seemed a bit superfluous, as set(get()+x) already works fine and should be obvious.

Reply:
Seeing I wasn't the only one with that idea: Done.